047_h_L552/047_h_L552_unsafe.py

```
@@ -0,0 +1,1176 @@
+            inp = input(bold("{prompt}> ".format(prompt=os.getcwd()))).strip()
+        except (KeyboardInterrupt, EOFError):
+            print()
+            my_exit(0)
+        if len(inp) == 0:
+            continue
+        if inp in ("h", "help()"):
+            info()
+        elif inp in ("q", "qq", ":q", ":x", "quit()", "exit()"):
+            my_exit(0)
+        elif inp in ("c", "clear()"):
+            os.system("clear")
```

Ruff (0.14.1)

Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)